System/SwatEquipment.ini

```diff
@@ -4089,7 +4089,7 @@ PlayerUsable=true
 EnemyUsesAmmo=(AmmoClass="SwatAmmo.NovaPumpSGAmmo",Chance=60)
 EnemyUsesAmmo=(AmmoClass="SwatAmmo.NovaPumpSG000Ammo",Chance=30)
 EnemyUsesAmmo=(AmmoClass="SwatAmmo.NovaPumpSGSabotAmmo",Chance=10)
-MagazineSize=7
```

Comment:
The website says 4+1

Reply:
The nova has an extended tube with 7+1, but swat 4 doesn't have it modeled.